core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -692,7 +692,7 @@ protected function findCurrentId(?string $table): ?string
 		$mode = Input::get('mode');
 
 		// For these actions the id parameter refers to the parent record
-		if (($act === 'paste' && $mode === 'create') || \in_array($act, array(null, 'select', 'editAll', 'overrideAll', 'deleteAll'), true))
+		if ($act === 'paste' || \in_array($act, array(null, 'select', 'editAll', 'overrideAll', 'deleteAll'), true))
```

Comment:
```suggestion
		if (\in_array($act, array(null, 'paste', 'select', 'editAll', 'overrideAll', 'deleteAll'), true))
```

If we decide to merge this PR, this should go into the in_array check I think.